services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -214,7 +212,7 @@ def _column_from_feature(
         dataset_feature_name: str, dataset_feature: Union[dict[str, Any], list[Any]]
     ) -> Optional[SupportedColumns]:
         if isinstance(dataset_feature, list) or (
-            isinstance(dataset_feature, dict) and dataset_feature.get("_type") in ("LargeList", "Sequence")
+            isinstance(dataset_feature, dict) and dataset_feature.get("_type") in ("LargeList", "List", "Sequence")
```

Comment:
why do we keep testing "Sequence" here?

Reply:
because of existing datasets: in the dataset-viewer cache they are Sequence